README.md

```diff
@@ -417,6 +417,28 @@ If you're not allowing users to log in with passwords, you may disable the
 username/password form from Django admin login page by setting `HELUSERS_PASSWORD_LOGIN_DISABLED`
 to `True`.
 
+
+#### Migrating old AD user from Tunnistamo to Keycloak
+
+When transitioning from one AD authentication to another, it is possible to migrate
+the old user data for the new user with a different UUID. Migration is done by finding
+the old user instance and replacing its UUID with the new one from the token
+payload. Migration happens one user at a time upon login.
+
+Feature can be configured using the following settings.
+- `HELUSERS_AD_MIGRATE_ENABLED`: Enable the feature. Defaults to `False`.
+- `HELUSERS_AD_MIGRATE_EMAIL_DOMAINS`: Whitelisted email domains for migration.
+  Defaults to `["@hel.fi"]`.
```

Comment:
Technically `@` is not part of a domain, and now these are actually "email ends with" instead of "email domain". So I wonder if it would be less error-prone to have only the actual domains here in this setting (`["hel.fi"]`), and force that by comparing those against the part after `@`. 🤔